lib/parliament/request.rb

```diff
@@ -22,7 +22,9 @@ def respond_to_missing?(method, include_private = false)
     end
 
     def get
-      Net::HTTP.get(URI(api_endpoint))
+      data = Net::HTTP.get(URI(api_endpoint))
```

Comment:
Do we need to add error checking around this?

What happens if the API goes down? Or there's a 500 error?